backend/db/items.js

```diff
@@ -4,7 +4,7 @@ const client = require('./client')
 
 async function adminCreateItem ({ name, category, brand, size, price, image, description, inventory }) {
     try {
-        const { rows: [item] } = await client.query(`
+        const { rows: {item} } = await client.query(`
```

Comment:
Tested this change in isolation on my branch, causing the seed to fail when creating items